vmm/src/signal.rs

```diff
@@ -0,0 +1,230 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I'm not really keen on having _linux signals related code_ split around different crates like so. For example there is also `sys_util/src/signal.rs`.

The logic looks solid :+1: